### PR TITLE
Notification enhancements

### PR DESCRIPTION
**Pull request by btry**
Cleanup notification implementation

### CODE REVIEW

**Comment by btry** [comment]
rebased, needs to review if it is enough unit tested

File: inc/notificationtargetaccountvalidation.class.php [hunk 1]
```diff
@@ -61,19 +61,15 @@ static function getTypeName($nb=0) {
     */
    public function getEvents() {
       return array(
-          self::EVENT_SELF_REGISTRATION => __('User registration', 'flyvemdmdemo')
+            self::EVENT_SELF_REGISTRATION          => __('User registration', 'flyvemdmdemo'),
+            self::EVENT_TRIAL_BEGIN                => __('Start of trial period', 'flyuvemdmdemo'),
+            self::EVENT_TRIAL_EXPIRATION_REMIND_1  => __('First expiration reminder', 'flyuvemdmdemo'),
+            self::EVENT_TRIAL_EXPIRATION_REMIND_2  => __('Second expiration reminder', 'flyuvemdmdemo'),
+            self::EVENT_POST_TRIAL_REMIND          => __('post-trial reminder', 'flyuvemdmdemo'),
```
Comment (ajsb85) [inline inc/notificationtargetaccountvalidation.class.php]: "Post-trial"
May first char uppercase.

**Review by ajsb85 — APPROVED** [review]
Reviewed

File: inc/notificationtargetaccountvalidation.class.php (hunk 1 above)
Comment (btry) [reply inc/notificationtargetaccountvalidation.class.php]: +1
and an other typo on all visible lines here by the way

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Pull request merged**